src/main/scala/pipelines/images/cifar/RandomPatchCifarAugmented.scala

```diff
@@ -99,7 +99,7 @@ object RandomPatchCifarAugmented extends Serializable with Logging {
     val testPredictions = predictionPipeline(testImagesAugmented)
 
     val testEval = AugmentedExamplesEvaluator(
-      testImageIdsAugmented, testPredictions, testLabelsAugmented, numClasses)
+      testImageIdsAugmented, testPredictions.get, testLabelsAugmented, numClasses)
```

Comment:
Minor: But we could make AugmentedExamplesEvaluator take in two PipelineDatasets instead of one PipelineDataset and RDD